oxidation/libparsec/crates/types/src/addr.rs

```diff
@@ -805,10 +805,10 @@ impl BackendInvitationAddr {
         let base = BaseBackendAddr::from_url(&parsed)?;
         let organization_id = extract_organization_id(&parsed)?;
         let pairs = parsed.0.query_pairs();
-        let invitation_type = match extract_action(&pairs)? {
-            x if x == "claim_user" => InvitationType::User,
-            x if x == "claim_device" => InvitationType::Device,
-            x if x == "claim_shamir_recovery" => InvitationType::ShamirRecovery,
+        let invitation_type = match &extract_action(&pairs)?[..] {
```

Comment:
```suggestion
        let invitation_type = match extract_action(&pairs)?.as_ref() {
```

Reply:
why? plus this requires importing [lazy_static](https://rust-locale.github.io/rust-locale/lazy_static/index.html)::[__Deref](https://rust-locale.github.io/rust-locale/lazy_static/trait.__Deref.html)